python/lsst/ts/xml/generate_subsystems_doc.py

```diff
@@ -294,9 +294,13 @@ def generate_subsystems_doc():
                         state_name = states[0].split("_")[0].lstrip()
                         cf.write(f":{state_name}:\n")
                         for state in states:
-                            cf.write(
-                                f"  * :any:`{state.split('_')[1]} <lsst.ts.idl.enums.{subsystem}.{state_name}.{state.split('_')[1].upper()}>`\n"  # noqa
-                            )
+                            # state format starts as [\n statename_state ]
+                            state = state.split("_")[
+                                1:
+                            ]  # Some states have more than one underscore split.
+                            state = "".join(state)  # Make that list a string
+                            state = state.lstrip()  # Strip the new line
```

Comment:
Please delete this line.

If you split "\n foo_bar_baz" with `state = state.split("_", 1)[1] ` you end up with `state = "bar_baz"`. The \n is already gone because it was part of the first piece of the split that you threw away: the discarded string is `"\n foo"`.